packages/webdriverio/tests/addCommand.test.js

```diff
@@ -182,6 +185,61 @@ describe('addCommand', () => {
             expect(typeof otherSubElem.myCustomElementCommand).toBe('function')
             expect(await otherSubElem.myCustomElementCommand()).toBe('foobar-.otherSubElem')
         })
+
+        test('should properly throw exceptions on the browser scope', async () => {
+            const browser = await remote(remoteConfig)
+            browser.addCommand('function1', function () {
+                throw error1
+            })
+
+            browser.addCommand('function2', function () {
+                browser.$('#foo')
+                throw error2
+            })
+
+            await expect(() => browser.function1()).toThrow(error1)
+            await expect(() => browser.function2()).toThrow(error2)
+
+            try {
+                await browser.function1()
+            } catch (error) {
+                expect(error).toBe(error1)
```

Comment:
there is a chance that this assertion is never being run if the try statement actually runs successfully. Therefor I suggest to put this into a new `test` block and use something like `expect.assertions(number);`

Reply:
done.